### ISSUE TITLE
communities: support for multiple curators per community

### ISSUE THREAD

**Issue by lnielsen** [zenodo/zenodo#810]

**Comment by eldemet** [comment]
I would like to give a 👍  on this issue, hopefully it will be addressed in a future milestone.

**Comment by behrica** [comment]
As well important for us from EFSA.
For us the community is in reality "EFSA owned", not really "personell".
So it would be very usefull, if there could be multiple curators or goups per community.

**Comment by duvivier** [comment]
We also would appreciate the ability to have multiple curators for a community.

**Comment by moble** [comment]
Even if only one user is allowed to curate, is there at least a way to *transfer* curatorship from one user to another?  If someone is leaving the field or the group, or just gets tired of the responsibility, it would be nice to be able to transfer it without the curator just giving the password to someone else.

**Comment by bearore** [comment]
is any update available on this? would be great to have multiple curators for a community

**Comment by mrustl** [comment]
Is this topic still open? Would be really appreciated if multiple curators are supported!

**Comment by zaneselvans** [comment]
Having more than one curator seems key to creating a community -- otherwise it seems kind of dictatorial. We want to be able to have more than one person in our organization curate our organizational data products, rather than having them tied to a specific individual.

**Comment by buddhasystem** [comment]
Has this been addressed? Thank you!

**Comment by monicacecilia** [comment]
Hi again dear Zenodo team, 
Are there any updates about this request? 
Is this being considered as a possibility? Not in a million years? :bowtie: 
.... I'm just a hopeful user wondering about the same things others have described above. 
Hearing any updates would be brilliant!
Thank you!

**Comment by florianthiery** [comment]
same here... thx :-)

**Comment by JuliaR2Punkt0** [comment]
Are there any updates on this? I think in a community it should be possible to curate as a community. Thank you!

**Comment by edzob** [comment]
Is there a way we can help in creating this functionality?

**Comment by christofs** [comment]
+1 - would be very useful for us, too.

**Comment by carlosvega** [comment]
This would be great

**Comment by tpaixao** [comment]
At the very least, the ability of changing the email address of the curator, would be great

**Comment by buddhasystem** [comment]
This issue has a huge impact on usability. It's been on the wish list for about 2 years, if I remember correctly. From the community perspective, this is issue number one with Zenodo.

**Comment by edzob** [comment]
is there anyway that we as a community help with code, documentation or other tasks?

**Comment by nuest** [comment]
[Note: I'm not affiliated with either Zenodo or Invenio, this is just an outsider's guess!]

My guess is that this will not be realised for Zenodo any time soon, but will happen when Zenodo is switched over to InvenioRDM, some time in 2021. Looking at the code base for `invenio-communities`, the member API seems to define different roles and mechanisms to invite new members and assign roles: https://github.com/inveniosoftware/invenio-communities/blob/master/invenio_communities/members/models.py

@lnielsen maybe you can confirm?

**Comment by tpaixao** [comment]
I solved my problem by writing directly to Zenodo's support team. They can change the user account that manages a community, just need confirmation from the recipient that they accept. It's not the most streamlined solution, but it helps until this gets implemented.
Leaving it here, as it may help others.

**Comment by lnielsen** [comment]
@nuest Correct. Multiple curators support is being implemented as part of InvenioRDM and the core features are already implemented. We're planning to have it tidied up by April/May (see https://inveniosoftware.org/products/rdm/roadmap/).

You can help out by joining the InvenioRDM project. This is where most of the coding is happening right now.

**Comment by abubelinha** [comment]
I am surprised nobody has mentioned an obvious and frequent problem these days: 
What if a community curator dies?

We never know if she/he will have internect connection in order to write directly to Zenodo's support team from the other side.

**Comment by cntanos** [comment]
The bare minimum here is to have the ability to change the curator. Then, having mutliple curators is the simplest thing you can do to plan for contingencies. Handing over responsibilities is a normal practice. Someone may have left a company, switched projects, the responsible party may have changed, they may have died etc. Communities shouldn't be linked to a single person forever.

The longer you postpone this solution, the more you are encouraging **password sharing** for personal accounts. This is a **major security risk**. This should be **highly prioritised**.

**Comment by dagewa** [comment]
I'm waiting for this feature too. I want my community to be maintained by a few individuals from diverse labs for neutrality, not just by me. Also I have occasionally missed requests while on holiday. I really need to have a team of curators!

**Comment by lnielsen** [comment]
It's implemented and available on InvenioRDM (see https://inveniordm.docs.cern.ch/releases/versions/version-v9.0.0/#members). You can test is out on https://inveniordm.web.cern.ch, and feel free to provide feedback.  

We are focused with all efforts to move Zenodo on top of InvenioRDM. This move is critical for Zenodo's long-term sustainability, since it allows co-developing the technical platform for Zenodo with many other partners. Please bear with us for a while longer, as we're doing what we can with our available resources. We'll be communicating more about the project to move and the timelines in Aug/Sep.

**Comment by dagewa** [comment]
NB from https://blog.zenodo.org/2022/12/07/2022-12-07-zenodo-on-inveniordm/:
> **Timeline**
>
>We are expecting to go-live with the new platform in Autumn 2023.

**Comment by Luckycat8906** [comment]
Hello, 

Is there any new information on this?  
Are we already able to add curators, owners etc?

**Comment by dagewa** [comment]
It looks like it is possible to invite other members with a variety of roles:

![image](https://github.com/zenodo/zenodo/assets/11502083/777c24fc-0a89-4276-bb3f-59efae7cfda3)

I've not actually done it yet because I'm waiting for confirmation of the right username to add.

**Comment by cntanos** [comment]
Yes. Finally it was implemented and the new version of Zenodo looks really nice. For the time being, though, inviting another member is done only by searching them with the username, because they can't be found with their email. A useful feature would also be for people to ask to join a community, instead of only being invited to it.

**Comment by joshmoore** [comment]
:clap:

**Comment by Luckycat8906** [comment]
Great 💯  thanks everybody for updating.

**Comment by floswald** [comment]
Hi all,
I logged a feature request with the support team a while ago: I am looking for an "assitant curator" role. they would be able to see community inclusion requests, check their content, talk to the main curator about it etc - but they could NOT accept the request. Only the curator can. I need this workflow because I have a large number of assistants checking replication packages for a journal but I do not want to grant them the right to accept the packages. thanks for any updates!

![Screenshot 2024-04-03 at 15 08 31](https://github.com/zenodo/zenodo/assets/1021558/53e64d8b-81a8-448e-9af4-7d820184eb16)
